Resources/Prototypes/ai_factions.yml

```diff
@@ -113,4 +120,4 @@
   - Dragon
   - WizFedFaction # Frontier
   - BloodCultNF # Frontier
-  - DinosaursNF # Frontier
+  - StreetGangNF # Frontier
```

Comment:
Revs should still find dinos hostile, right?  Looking above, still true the other way around.

Reply:
I probably made a mistake when I was resolving merge conflicts. Gotta take a careful look

Reply:
And while I'm at it, might as well add factions for the rest on newly added expedition enemies

Reply:
Pushed fix. Hopefully I made the ai_faction files more readable